### PR TITLE
[denonmarantz] Extend to include Denon DRA-100 sources

### PR DESCRIPTION
**Pull request by kaikreuzer**
See https://community.openhab.org/t/contributing-without-github-account-add-new-sources-for-denon-dra-100/148283.

I am not familiar enough with Denon AVRs and whether the DRA-100 should constitute a different thing type or not.

So I leave the decision whether this should be merged as is to the code owner @jwveldhuis.

### CODE REVIEW

**Comment by jlaur** [comment]
> I am not familiar enough with Denon AVRs and whether the DRA-100 should constitute a different thing type or not.

It should not. There is only one common `avr` thing that can be used with any model.

The list of input sources constitutes a union of input sources for all models. For example, in the list was already `BT`, which is the Bluetooth input source for some models. For DRA-100 this is named `BLUETOOTH`. Likewise, the oldest models have `NET/USB` while newer models have them separated as `NET` and `USB`.

In other words - everything looks fine. Thanks for picking up the patch and making it into a PR. 👍

**Review by jlaur — APPROVED** [review]
LGTM
